backend/laundry/management/commands/update_laundry_rooms.py

```diff
@@ -0,0 +1,66 @@
+import csv
```

Comment:
what is this new file for? Can you describe the changes between the new API and old API in the PR description for more context about what you are doing?